doc/changes/latest.inc

```diff
@@ -37,6 +37,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug in convert_flash_mris, when flash30 and no flash5_reg.mgz file regenerated, add missing_ok=True as it is done in line 2074. (:gh:`11842` by :newcontrib: `Hamza Abdelhedi`)
```

Comment:
Hello, the change log should help users understand what changes for them. Please describe how the behavior changed from a user's perspective, not what you did as a developer

Reply:
@hoechenberger done

Reply:
FYI @hoechenberger the space between the `:newcontrib:` and the backticks here made the role not work... fixing in https://github.com/mne-tools/mne-python/pull/11837

Reply:
Oh.. thanks!